lms/djangoapps/instructor/views/tools.py

```diff
@@ -292,8 +294,9 @@ def generate_course_forums_d3(url_handle):
             previous[1] = number
         elif comment_type == 'Comment':
             previous[2] = number
+
     # return nothing if there's no data
-    if len(rows) == 1:
+    if not previous:
```

Comment:
I'm not 100% sure; I had assumed that's what Jim meant by his "Also fixed a bug when there is only 1 row" comment.